MAUI/Maps/tooltip.md

```diff
@@ -0,0 +1,468 @@
+![Maps bubble tooltip builder](images/tooltip/bubble_tooltip_builder.png)
+
+## Tooltip for the markers
+
+The [`MapShapeLayer.ShowMarkerTooltip`](https://pub.dev/documentation/syncfusion_flutter_maps/latest/maps/MapShapeLayer/
+shapeTooltipBuilder.html) is used to clearly indicate the marker information every time when you interacts with the markers i.e., 
+while tapping in touch devices and hover enter in the mouse enabled devices.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+     <map:SfMaps>
+            <map:SfMaps.Layer>
+                <map:MapShapeLayer 
+                        ShapesSource="https://cdn.syncfusion.com/maps/map-data/world-map.json"  ShapeStroke="DarkGrey" ShowMarkerTooltip="True">
+
+                    <map:MapShapeLayer.Markers>
+                        <map:MapMarkerCollection>
+                            <map:MapMarker
+                              Latitude="1454.6" IconWidth="20" IconHeight="20" IconType="Diamond" 
+                                 Longitude="36.0"  />
+                            <map:MapMarker
+                              Latitude="34.0479" IconWidth="20" IconHeight="20" IconType="Circle" 
+                                 Longitude="100.6124" />
+
+                        </map:MapMarkerCollection>
+                    </map:MapShapeLayer.Markers>
+           </map:SfMaps.Layer>
+    </map:SfMaps>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+public MainPage()
+	{
+		InitializeComponent();
+        SfMaps maps = new SfMaps();
+        MapShapeLayer layer = new MapShapeLayer();
+
+        layer.ShapesSource = MapSource.FromUri(new Uri("https://cdn.syncfusion.com/maps/map-data/world-map.json"));
+        layer.ShowMarkerTooltip = true;
+        MapMarker diamondMarker = new MapMarker();
+        diamondMarker.Longitude = 36.0;
+        diamondMarker.Latitude = 1454.6;
+        diamondMarker.IconHeight = 20;
+        diamondMarker.IconWidth = 20;
+        diamondMarker.IconType = MapIconType.Diamond;
+
+        MapMarker circleMarker = new MapMarker();
+        circleMarker.Longitude = 100.6124;
+        circleMarker.Latitude = 34.0479;
+        circleMarker.IconHeight = 20;
+        circleMarker.IconWidth = 20;
+        circleMarker.IconType = MapIconType.Circle;
+
+        MapMarkerCollection mapMarkers = new MapMarkerCollection();
+        mapMarkers.Add(diamondMarker);
+        mapMarkers.Add(circleMarker);
+
+        layer.Markers = mapMarkers;
+        maps.Layer = layer;
+        this.Content = maps;
+    }
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Maps marker tooltip builder](images/tooltip/marker_tooltip_builder.png)
+
+N>
+* Refer the [`MapTooltipSettings`](https://pub.dev/documentation/syncfusion_flutter_maps/latest/maps/MapTooltipSettings-class.html), for customizing the tooltip.
+
+## Appearance customization
+
+## Customizing Data Templates in ToolTip
+
+Data template can be customized for items and headers of the control. The following code example shows the usage of DataTemplate.
```

Comment:
Please refer to the below UG for descriptions

https://help.syncfusion.com/maui/listview/viewappearance

Reply:
Modified sir